src/hooks/useInitialStartup.js

```diff
@@ -162,7 +163,7 @@ export const useInitialStartup = () => {
     LogBox.ignoreAllLogs();
 
     // statusbar
-    if (Platform.OS === 'android') StatusBar.setBackgroundColor('#ffffff');
+    if (Platform.OS === 'android') StatusBar.setBackgroundColor(COLORS.white);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is also beneficial as it replaces the hard-coded color value with a dynamic one from the theme file. This will make it easier to manage and update colors in the future.
```diff
-    if (Platform.OS === 'android') StatusBar.setBackgroundColor('#ffffff');
+    if (Platform.OS === 'android') StatusBar.setBackgroundColor(COLORS.white);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->